test/subscriber.ts

```diff
@@ -648,33 +648,35 @@ describe('Subscriber', () => {
 
     beforeEach(() => {
       // Pre-define _tracing to gain access to the private field after subscriber init
-      tracingSubscriber['_tracing'] = undefined;
+      tracingSubscriber['_useOpentelemetry'] = false;
     });
 
     it('should not instantiate a tracer when tracing is disabled', () => {
       tracingSubscriber = new Subscriber(subscription);
-      assert.strictEqual(tracingSubscriber['_tracing'], undefined);
+      assert.strictEqual(tracingSubscriber['_useOpentelemetry'], false);
     });
 
     it('should instantiate a tracer when tracing is enabled through constructor', () => {
       tracingSubscriber = new Subscriber(subscription, enableTracing);
-      assert.ok(tracingSubscriber['_tracing']);
+      assert.ok(tracingSubscriber['_useOpentelemetry']);
     });
 
     it('should instantiate a tracer when tracing is enabled through setOptions', () => {
```

Comment:
I think this "instantiate a tracer" needs to be rephrased with this change. It will always instantiate a tracer, but it will be a noop

Reply:
originally i was thinking to just remove these tests 🙈

Reply:
I guess we could verify that it's a no-op tracer, but if that's the way OT works now, I'm also okay with just removing those tests.

Reply:
Yeah, that sounds reasonable. The unit test is using a in-memory trace provider / exporter to determine if the spans get created or not. I am just struggling a bit with mocha and how things are mocked. I can have a stab at it again next week. 

Would you be able with me just removing the whole opt-in settings flag `enableOpentelemetryTracing`?

Reply:
I was thinking about this too... the parts that call OT aren't really a problem, but currently it's adding a message attribute to things sent through Pub/Sub. I haven't dug into it enough to see if that gets removed again later or not, but it might increase a user's bandwidth or storage unintentionally. Maybe it can just be repurposed to enable those attributes for correlating on the server side.

Reply:
Good point, we probably shouldn't be doing that unless its a real trace AND opted into. You can check if its a real vs no-op trace with [`isSpanContextValid()`](https://github.com/open-telemetry/opentelemetry-js-api/blob/92c170009ba36041ec0adba7804ce0a06d1b0330/src/api/trace.ts#L78)

Reply:
Added it